server/scsynth/scsynth_main.cpp

```diff
@@ -51,7 +51,9 @@ void Usage() {
              "   -v print the supercollider version and exit\n"
              "   -u <udp-port-number>    a port number 0-65535\n"
              "   -t <tcp-port-number>    a port number 0-65535\n"
-             "   -B <bind-to-address>    an IP address\n"
+             "   -B <bind-to-address>\n"
+             "          Bind the UDP or TCP socket to this address.\n"
+             "          Default 127.0.0.1. Set to 0.0.0.0 to listen on all interfaces.\n"
```

Comment:
Does it make sense to put doublequotes around these addresses to indicate they are string values?